schema/updates/2022_11_21_SSX_experiment.sql

```diff
@@ -0,0 +1,124 @@
+INSERT INTO SchemaStatus (scriptName, schemaStatus) VALUES ('2022_11_21_SSX_experiment.sql', 'ONGOING');
+
+CREATE TABLE `SSXDataCollection`(
+    `dataCollectionId` int(11) unsigned PRIMARY KEY,
+    `repetitionRate` float DEFAULT NULL,
+    `energyBandwidth` float DEFAULT NULL,
+    `monoStripe` varchar(255) DEFAULT NULL,
+    `jetSpeed` float DEFAULT NULL,
+    `jetSize` float DEFAULT NULL,
+    `chipPattern` varchar(255) DEFAULT NULL,
+    `chipModel` varchar(255) DEFAULT NULL,
+    `reactionDuration` float DEFAULT NULL,
+    `laserEnergy` float DEFAULT NULL,
+    `experimentName` varchar(255) DEFAULT NULL,
+    FOREIGN KEY (`dataCollectionId`) REFERENCES `DataCollection`(`dataCollectionId`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE = InnoDB AUTO_INCREMENT = 0 DEFAULT CHARSET = utf8mb4;
+
+ALTER TABLE
+    `DataCollectionGroup`
+MODIFY
+    COLUMN `experimentType` enum(
+        'EM',
+        'SAD',
+        'SAD - Inverse Beam',
+        'OSC',
+        'Collect - Multiwedge',
+        'MAD',
+        'Helical',
+        'Multi-positional',
+        'Mesh',
+        'Burn',
+        'MAD - Inverse Beam',
+        'Characterization',
+        'Dehydration',
+        'Still',
+        'SSX-Chip',
+        'SSX-Jet'
+    );
+
+ALTER TABLE
+    `ComponentType`
+ADD
+    UNIQUE (`name`);
+
+INSERT INTO
+    `ComponentType` (`name`)
+values
+    ('Ligand'),
+    ('Buffer'),
```

Comment:
At Diamond, we have Buffer in the `ComponentSubType` table, which has a m-n relationship to `Protein` so that a protein can have multiple subtypes. I guess you can have multiple components here as well, although they are linked to `Crystal` and `BLSample` via `CrystalComposition` and `SampleComposition`. So, probably not applicable here, but thought I should mention it.